Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-HAFNIUM_-_Exchange_0-day_exploits_README.md

```diff
@@ -0,0 +1,41 @@
+This playbook includes the following tasks:
+
+- Collect indicators to be used in your threat hunting process
+- Retrieve IOCs related to HAFNIUM and the exploited exchange 0-day vulnerabilities
+- Discover IOCs related to the attack
+- Query firewall logs to detect malicious network activity
+- Search endpoint logs for malicious hashes to detect compromised hosts (Available from Cortex XSOAR 5.5.0).
+- Block indicators
+
+Read more about the attack on our Unit42 blog: https://unit42.paloaltonetworks.com/microsoft-exchange-server-vulnerabilities/"
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Block Indicators - Generic v2
+* Search Endpoint by CVE - Generic
+* Search Endpoints By Hash - Generic V2
+* Palo Alto Networks - Hunting And Threat Detection
+
+### Integrations
+This playbook does not use any integrations.
```

Comment:
@altmannyarden I think this is mis-leading. We should add here the optional integrations used by the sub playbooks or simply remove this section.